### ISSUE TITLE
Connman Does Not Seem to Be Respecting or Correctly Using the 'ResolvConf' Configuration Setting

### ISSUE THREAD

**Issue by gerickson** [Nuovations/connman#4]
Despite having set `ResolvConf=/dev/null` per the man page, `connmand` seems to be unhappy with and not following the code path for `ResolvConf=/dev/null`:

```
   Jul 12 18:02:42 19205453 daemon.warn connmand[360]: Cannot create /var/run/connman/resolv.conf falling back to /etc/resolv.conf
```

While I realize this is a harmless warning, it seems at odds with the man page and the code path for `resolvfile_export`.

I’ve manually-created a static, read-only /etc/resolv.conf which need not be dynamically updated at runtime:

```
   nameserver 127.0.0.1
   nameserver ::1
```

**Comment by gerickson** [comment]
Nicolas Gariepy has submitted a [patch to the connman mailing list that should address this issue](https://lore.kernel.org/connman/344155DA-16CE-4CC8-847E-CA3EBB00A664@nuovations.com/T/#t).

**Comment by gerickson** [comment]
Closed with #19.
